### PR TITLE
[master] Common `results` notation for api results

### PR DESCRIPTION
**Pull request by leonkozlowski**
* common results notation for api results

* bump version

### CODE REVIEW

File: betfund_bet365/response/pre_match_odds.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def __init__(self, data):
         super(PreMatchOddsResponse, self).__init__(data)
 
     @property
-    def events(self) -> Union[List[FiResultBase], None]:
+    def results(self) -> Union[List[FiResultBase], None]:
```
Comment (mitchbregs) [inline betfund_bet365/response/pre_match_odds.py:54]: Why the change here?

**Review by mitchbregs — APPROVED** [review]
This LGMT, just wondering why `events` -> `results`.  Feel like `events` is more appropriate, no?

File: betfund_bet365/response/pre_match_odds.py (hunk 1 above)
Comment (leonkozlowski) [reply betfund_bet365/response/pre_match_odds.py:54]: In order to standardize downstream usage of this wrapper I am making response object top level structure common for all objects.

**For Example**
-All responses contain two common keys:

```python
{
    "success": 1,
    "results": []
}
```

I was originally calling two of them `"events"` for some reason -- bringing these all to `"results"` because downstream caller will use one helper to ingest results object.

see: [Previous Pull Request](https://github.com/betfund/betfund-bet365/pull/2)

Comment (mitchbregs) [reply betfund_bet365/response/pre_match_odds.py:54]: Beautiful, totally makes sense.
